### DISCUSSION TITLE
Do we have a clear step by step guide to adding wrap files to WrapDB?

### DISCUSSION THREAD

**Discussion by dreamer-coding** [mesonbuild/meson#13187] [General]
Do we have a clear step by step guide to adding wrap files to WrapDB? I had added some wrap files before but not sure how the new system works. At some point I'd like to add Fossil Logic libraries to the list. All of them are built with Meson as the primary tool.

One of the Fossil Logic components: 
https://github.com/fossil-lib/fscl-xstring-c

**Comment by bruchar1** [comment]
https://mesonbuild.com/Adding-new-projects-to-wrapdb.html
